walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/images2.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Ishika's Image Gallery</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div id="content">
+      <h1>Ishika's Image Gallery</h1>
+      <div class="image-thumbnails">
+        <a href="images2/Genn.jpg"><img src="images2/Genn.jpg"/></a>
+        <a href="images2/Sheryl.jpg"><img src="images2/Sheryl.jpg"/></a>
+        <a href="images2/Sun.jpg"><img src="images2/sun.jpg"/></a>
+        <a href="images2/book.jpg"><img src="images2/book.jpg"/></a>
+      </div>
+    </div>
+  </body>
+</html>
```

Comment:
nit: Add a newline. Always a good idea to have an empty newline at end of file.
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline